content/applications/inventory_and_mrp/manufacturing/quality_control/instructions_check.rst

```diff
@@ -0,0 +1,85 @@
+created. Alternatively, a |QCP| can be configured that will automatically create checks at a
+predetermined interval.
+
+This documentation will only detail the configuration options and processing steps that are unique
+to Instructions quality checks. For a full overview of all the configuration options available when
+creating a single check or a |QCP|, see the documentation on :ref:`Quality checks
+<manufacturing/quality_control/quality-checks>` and :ref:`Quality Control Points
+<manufacturing/quality_control/quality-control-points>`.
+
+Create an Instructions quality check
+====================================
+
+Instructions quality checks can be created by navigating to :menuselection:`Quality --> Quality
+Control --> Quality Checks`, and clicking :guilabel:`New`. It is also possible to create a |QCP|
+that is configured to automatically create an Instructions quality check when
+certain criteria is met. To do so, navigate to :menuselection:`Quality --> Quality Control -->
+Control Points`, and click :guilabel:`New`.
+
+On the quality check or |QCP| form, select :guilabel:`Instructions` from the :guilabel:`Type`
+drop-down field. If a single quality check is being created, enter the instructions for how to
+complete the check in the :guilabel:`Instructions` text entry field of the :guilabel:`Notes` tab at
+the bottom of the form. If a |QCP| is being configured, enter the instructions for how to complete
+the checks it creates in the text-entry field of the :guilabel:`Instructions` tab at the bottom of
+the form.
+
+Fill out the rest of the required fields for the quality check or |QCP|, as detailed in the
+documentation for :ref:`Quality checks <manufacturing/quality_control/quality-checks>` and
+:ref:`Quality Control Points <manufacturing/quality_control/quality-control-points>`. Finally, click
+:guilabel:`Save` to create the quality check or |QCP|.
+
+Process an Instructions quality check
+=====================================
+
+To process an Instructions quality check, select a manufacturing order or an inventory order
+(receipt, delivery, return, etc.). Manufacturing orders can be selected by navigating to
+:menuselection:`Manufacturing --> Operations --> Manufacturing Orders`, and clicking on an order.
+Inventory orders can be selected by navigating to :menuselection:`Inventory`, clicking the
+:guilabel:`# To Process` button on an operation card, and selecting an order.
+
+On the selected inventory or manufacturing order, a purple :guilabel:`Quality Checks` button appears
+at the top of the order if the order has quality checks that need to be completed. Click the button
+to open the :guilabel:`Quality Check` pop-up window, which shows all of the quality checks required
+for that order.
+
+.. image:: instructions_check/quality-check-pop-up.png
+   :align: center
+   :alt: The Quality Check pop-up window on a manufacturing or inventory order.
```

Comment:
I think this image can benefit from even more explanation and context around Instructions quality checks and how it adds to manufacturing. I'd first describe the UI and specifically call out names and references in the image in an example block and explain what the quality check aims to do.

```suggestion

.. example::
   In the :guilabel:`Quality Check` pop-up window of a manufacturing order for `FURN_9666 Table`,
   quality check `QC_00049` details textual instructions to ensure all table legs are attached to
   the table top.

   .. image:: instructions_check/quality-check-pop-up.png
      :align: center
      :alt: The Quality Check pop-up window on a manufacturing or inventory order.
```

Feel free to reword/ignore (: